utils/cliutils/commandsflags.go

```diff
@@ -1283,6 +1284,14 @@ var flagsMap = map[string]cli.Flag{
 		Name:  RequirementsFile,
 		Usage: "[Optional] [Pip] Defines pip requirements file name. For example: 'requirements.txt' ` `",
 	},
+	WithFixVersion: cli.BoolFlag{
+		Name:  WithFixVersion,
+		Usage: "[Optional] Set to true if you wish to display the audit results with fix version only ` `",
```

Comment:
...display the audit results with fix version only...
-->
...display issues which have a fix version only